modules/renter/hostdb/hostweight.go

```diff
@@ -47,18 +47,11 @@ var (
 
 	// requiredStorage indicates the amount of storage that the host must be
 	// offering in order to be considered a valuable/worthwhile host.
-	requiredStorage = func() uint64 {
-		switch build.Release {
-		case "dev":
-			return 1e6
-		case "standard":
-			return 5e9
-		case "testing":
-			return 1e3
-		default:
-			panic("incorrect/missing value for requiredStorage constant")
-		}
-	}()
+	requiredStorage = build.Select(build.Var{
+		Standard: uint64(20e9),
+		Dev:      uint64(1e6),
+		Testing:  uint64(1e3),
+	}).(uint64)
```

Comment:
required storage raised to 20GB, meaning to escape the storage penalty altogether you need to have 4 TiB of data.